resources/css/responsive.css

```diff
@@ -0,0 +1,53 @@
+@media only screen and (max-width: 1200px){
+  .container{
+    max-width: 998px;
+  }
+}
+
+@media only screen and (max-width: 768px){
+
+  .contentSection{
+    display: flex;
+    height: 1500px;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  .leftSection,
+  .middleSection,
+  .rightSection{
+    width: 100vw;
+  }
+
+  .leftSection,
```

Comment:
Please use hyphen separated class names instead of camel case.

Reply:
Entire responsive.css file has been edited